backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapper.kt

```diff
@@ -19,7 +19,6 @@
 package net.skyscanner.backpack.compose.cardwrapper
 
 import androidx.compose.foundation.BorderStroke
-import androidx.compose.foundation.border
```

Comment:
Why this file was changed?

Reply:
I think I might have gotten a warning about an unused import and deleted it here. Can I leave it as is or is it better to revert the change back to how it was?

Reply:
I think it's fine, as long as we awared